common/src/main/java/io/homeassistant/companion/android/database/AppDatabase.kt

```diff
@@ -527,6 +533,99 @@ abstract class AppDatabase : RoomDatabase() {
             }
         }
 
+        @RenameColumn.Entries(
+            RenameColumn(
+                tableName = "Authentication_List",
+                fromColumnName = "Username",
+                toColumnName = "username"
+            ),
+            RenameColumn(
+                tableName = "Authentication_List",
+                fromColumnName = "Password",
+                toColumnName = "password"
+            ),
```

Comment:
SQLite is case insensitive, but if you're going to rename columns here please also update the relevant data object.